### DISCUSSION TITLE
Thanks

### DISCUSSION THREAD

**Discussion by gitdrug** [jely2002/youtube-dl-gui#210] [General]
Thanks for making this program your super cool its relly usefull  (thats you a cool guy >😎😎)

**Comment by jely2002** [comment]
Thanks a lot for the kind words man, I appreciate it :)

**Reply by guyster104** [reply]
I 100% agree here. I used to use youtube-dlg to accomplish the task of downloading from youtube, but this app is 100% more accessible. I am totally blind, use a screen reader called JAWS, and am able to navigate every control. I would love to try it in the linux environment with Orca. Is there a .deb package available to install youtube-gui?

**Reply by TheGous** [reply]
Count me in, best Software right now, tried others and they are pure Trash, thanks! :D

**Comment by Chris-toph** [comment]
Yes, your soft is just perfect. Thanks a lot!
Is there a difference to install Open Video Downloader via the microsoft cern store or from the https://jely2002.github.io/youtube-dl-gui/ ?
Would the soft be affected with the ending of third-party cookies that google is trying to push forward?
Cheers,
C.

**Reply by jely2002** [reply]
Currently the Microsoft store version is a couple of versions behind. To always have the latest version, it's best to use the GitHub version.
I'm not sure about the 3rd party cookies, but I don't think it would affect this software much.

**Comment by aesculusglabra** [comment]
I'm quite fond of your app. Are there any plans to make it work natively with Apple Silicon?

**Reply by jely2002** [reply]
Thanks for the kind words. I don't have any apple machines to even test my application on. And making a VM is near impossible. So development for MacOS is really hard for me currently. My app uses ffmpeg and it's hard to find a silicon version of that, together with some other hiccups is making it a challenge to add native silicon support.

**Reply by aesculusglabra** [reply]
Totally understand – thanks for the response!
